cabal-install/Distribution/Client/ProjectOrchestration.hs

```diff
@@ -608,7 +616,7 @@ reportBuildFailures verbosity plan buildOutcomes
 
        -- For all failures, print either a short summary (if we showed the
        -- build log) or all details
-       die $ unlines
+       notice verbosity $ unlines
```

Comment:
This is a change in behavior. die unconditionally prints its output. notice can be suppressed. Maybe this is what you want but I get a bit worried that -v0 it will just die and you won't see any message.